scripts/_kxd

```diff
@@ -7,11 +7,11 @@ fi
 
 KUBECONFIG="$KUBECONFIG" _kxd_prompt
 
-selected_config="$HOME/.kube/$(cat ~/.kxd)"
+selected_config="$(cat ~/.kxd)"
 
 if [ -z "$selected_config" ]
```

Comment:
jbone pointed out this line was never getting hit because I had the kubeconfig dir in there. so the previous "default" config list item wasnt working anyway